app/perfsymboltable.cpp

```diff
@@ -336,7 +336,7 @@ int PerfSymbolTable::insertSubprogram(CuDieRangeMapping *cudie, Dwarf_Die *top,
     return locationId;
 }
 
-int PerfSymbolTable::parseDie(CuDieRangeMapping *cudie, Dwarf_Die *top, quint64 offset, quint64 size, quint64 relAddr, qint32 binaryId, qint32 binaryPathId,
+int PerfSymbolTable::parseDie(CuDieRangeMapping *cudie, Dwarf_Die *top, quint64 offset, quint64 size, quint64 relAddr, qint32 binaryId, qint32 binaryPathId, qint32 actualPathId,
```

Comment:
the actualPathId is unused here, but we must pass it to insertSubprogram, if you initialize the Symbol using {} you'll see that this is now broken

Reply:
I've fixed this all now locally